R/autoplot.pop_data.R

```diff
@@ -104,7 +111,9 @@ density_plot <- function(
   }
   if (log) {
     plot1 <- plot1 +
-      ggplot2::scale_x_log10() +
+      #ggplot2::scale_x_log10() +
+      ggplot2::scale_y_continuous(trans = scales::pseudo_log_trans(sigma = 0.01),
+                                  breaks = c(-1, -0.1, 0, 0.1, 1, 10), minor_breaks = NULL) +
```

Comment:
@d-morrison 
**scale_x_log10**
![2](https://github.com/UCD-SERG/serocalculator/assets/731279/8bd06bff-1962-4c4a-8525-441a99a7a3a4)

**scale_x_continuous**
![3](https://github.com/UCD-SERG/serocalculator/assets/731279/c5e2db00-3e7a-4fea-9a91-2cf2d7e1fff5)

Reply:
Thanks for these pictures; very helpful!

I'm realizing now that this change to the graph probably should have been a separate PR; but since we're already in the middle of things, let's just leave it in this PR.

I suggested a few fine-tuning changes to this graph in a patch PR: https://github.com/UCD-SERG/serocalculator/pull/128. @chrisorwa , please review!